index.bs

```diff
@@ -60,6 +60,7 @@ urlPrefix: https://tc39.github.io/ecma262/; spec: ECMASCRIPT
 </style>
 <script src="https://resources.whatwg.org/file-issue.js" async></script>
 <script src="https://resources.whatwg.org/commit-snapshot-shortcut-key.js" async></script>
+<link rel="manifest" href="/manifest.json">
```

Comment:
I'm a little skeptical of the value-add here. It's only used for "installation" purposes reportedly and all the metadata included is already readily available through other means in the document.